tutorials/MD/data/submit_em.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Let's see what ARC returns with, we might have to call the GPU partition here.

Reply:
Yes, submission scripts updated